CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Revision history for swarm
 
+## **0.2.0.0** - 2022-10-31
+
+A bunch of small fixes and improvements; special thanks to new
+contributors @0xcefaedfe, @kostmo, @ussgarci, and @valyagolev. Notable changes include:
+
+- New UI features:
+    - REPL improvements:
+        - Expose the last evaluated result as `it`, and previous results
+          as `itN` ([#734](https://github.com/swarm-game/swarm/pull/734))
+        - Allow clicking in the REPL input to move the cursor
+          ([#750](https://github.com/swarm-game/swarm/pull/750))
+        - Autocomplete entity names in the repl ([#798](https://github.com/swarm-game/swarm/pull/798))
+        - REPL cursor no longer blinks when REPL panel is not selected ([#801](https://github.com/swarm-game/swarm/pull/801))
+    - Improve user experience around quitting & moving between
+      tutorial challenges ([#754](https://github.com/swarm-game/swarm/pull/754))
+        - Add a button to the Quit dialog to restart a
+          scenario. ([#767](https://github.com/swarm-game/swarm/pull/767))
+        - Use scenario name as Goal dialog title ([#774](https://github.com/swarm-game/swarm/pull/774))
+    - `autoplay` flag for automatically demonstrating scenario
+      solutions ([#792](https://github.com/swarm-game/swarm/pull/792))
+    - Improved inventory sorting and user-controllable sort criteria ([#793](https://github.com/swarm-game/swarm/pull/793))
+    - Ability to temporarily hide robots so you can see what's under
+      them ([#802](https://github.com/swarm-game/swarm/pull/802))
+- New language features:
+    - New `void` type ([#735](https://github.com/swarm-game/swarm/pull/735))
```

Comment:
Huh, we forgot to add it to the editors because types change rarely and are not tested. 🤷

Reply:
Fixed! ✅